clang/runtime/dpct-rt/include/dpl_extras/algorithm.h.inc

```diff
@@ -1650,6 +1669,23 @@ inline void sort_keys(_ExecutionPolicy &&policy, key_t keys_in,
 }
 // DPCT_LABEL_END
 
+// DPCT_LABEL_BEGIN|sort_keys_io_iterator_pair|dpct
+// DPCT_DEPENDENCY_BEGIN
+// DplExtrasAlgorithm|sort_keys
+// DplExtrasIterators|io_iterator_pair
+// DPCT_DEPENDENCY_END
+// DPCT_CODE
+template <typename _ExecutionPolicy, typename key_t>
+inline void sort_keys(
+    _ExecutionPolicy &&policy, io_iterator_pair<key_t> &keys, int64_t n,
+    bool descending, int begin_bit = 0,
```

Comment:
`bool descending = false` to match the other overload of `sort_keys`?

Reply:
Good catch, fixed.